### PR TITLE
Prevent double submissions in forms

### PR DESCRIPTION
**Pull request by thivi**
### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: apps/console/src/extensions/configs/attribute.ts [hunk 1]
```diff
@@ -131,6 +131,6 @@ export const attributeConfig: AttributeConfig = {
         isSCIMCustomDialectAvailable: () =>  { return Promise.resolve(""); },
         isUserStoresHidden: () =>  { return Promise.resolve([]); },
         mapClaimToCustomDialect: false
-    }, 
-    showCustomDialectInSCIM: false
+    },
+    showCustomDialectInSCIM: true
```
Comment (brionmario) [inline apps/console/src/extensions/configs/attribute.ts:135]: Hope this change is intentional @thivi

Comment (thivi) [reply apps/console/src/extensions/configs/attribute.ts:135]: Yes. Setting this to `false` hid the custom attributes in the attribute mappings page. This has to be said to `true` to show them. @DimalChandrasiri, can you confirm if this is fine?

Comment (thivi) [reply apps/console/src/extensions/configs/attribute.ts:135]: I am reverting this to the previous state as we need to have a discussion on if this should be available here. If this should be set to `false`, then we should also disable the `Add Attribute Mapping` button in the Attributes pages.

Comment (thivi) [inline apps/console/src/extensions/configs/attribute.ts:135]:
```suggestion
    showCustomDialectInSCIM: false
```

**Pull request merged**